cfn-resources/autogen/schema-gen/main.go

```diff
@@ -412,21 +394,34 @@ func readRequestBody(method *openapi3.Operation, doc *openapi3.T) (schemaKeys []
 	return schemaKeys, reqSchema, definitions, requiredParams
 }
 
-func readResponseBody(method *openapi3.Operation, doc *openapi3.T) (schemaKey string, resSchema map[string]map[string]Property, definitions map[string]Definitions) {
-	var resSchemaKey string
+func readResponseBody(method *openapi3.Operation, openAPIDoc *openapi3.T) (schemaKey string, resSchema map[string]map[string]Property, definitions map[string]Definitions) {
+	if methodResponseHasSchema(method, "200") {
+		return readResponseBodyWithResponseCode(openAPIDoc, method, "200")
+	}
 
-	if method.Responses["200"] != nil && method.Responses["200"].Value != nil && method.Responses["200"].Value.Content["application/json"] != nil &&
-		method.Responses["200"].Value.Content["application/json"].Schema != nil {
-		resSchemaKey = filepath.Base(method.Responses["200"].Value.Content["application/json"].Schema.Ref)
-		// Read from Request body
-		if doc.Components.Schemas[filepath.Base(resSchemaKey)] != nil {
-			value := *doc.Components.Schemas[filepath.Base(resSchemaKey)]
-			resSchema, definitions = processSchema(resSchemaKey, &value, doc.Components.Schemas)
-		}
+	if methodResponseHasSchema(method, "201") {
+		return readResponseBodyWithResponseCode(openAPIDoc, method, "201")
+	}
```

Comment:
here if the **responseBody** does not have response 200, we check for response 201 (resource created)